_posts/2022-10-15-rtl.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: post
+title: Right to left scripting
+date: 2022-10-15 
+description: you can also write from right to left
+lang: fa
+tags: internationalization formatting math code
+category: internationalization
+---
+
+# عنوان
+این یک پست نمونه برای سنجش نوشتارهای راست‌به‌چپ است.
+
+
+## فرمول‌نویسی
+فرمول‌ها هم مانند کد به درستی به نمایش در‌خواهند آمد:
+
+
+$$
+\sum_{k=1}^\infty |\langle x, e_k \rangle|^2 \leq \|x\|^2
+$$
+
+## کد
+کدها هم‌چنان مانند قبل از چپ‌به‌راست خواهند بود.
+
+```python
+if lang=="rtl"
+    print("I am a right-to-left script!")
+```
```

Comment:
what's the purpose of this?

Reply:
It's merely a demonstration that shows code blocks stay left-to-right even if the language script is not. Now that I think of it, the code is a bit misleading... :)